app/code/Magento/Sitemap/Model/Sitemap.php

```diff
@@ -158,6 +163,13 @@ class Sitemap extends \Magento\Framework\Model\AbstractModel implements \Magento
     protected $_cacheTag = true;
 
     /**
+     * Last mode min timestamp value
+     *
+     * @var int
+     */
+    protected $lastModMinTsVal;
```

Comment:
Since this is a new property can we make it private.

Reply:
fixed

Reply:
fixed